File: src/main/java/leets/leenk/domain/leenk/domain/repository/LocationRepository.java
package leets.leenk.domain.leenk.domain.repository;

import leets.leenk.domain.leenk.domain.entity.Location;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface LocationRepository extends JpaRepository<Location, Long> {
}

File: src/main/kotlin/leets/leenk/domain/leenk/domain/repository/LocationRepository.kt
package leets.leenk.domain.leenk.domain.repository

import leets.leenk.domain.leenk.domain.entity.Location
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface LocationRepository : JpaRepository<Location, Long>
